dll/win32/kernel32/client/vista.c

```diff
@@ -39,7 +39,8 @@ SleepConditionVariableCS(IN OUT PCONDITION_VARIABLE ConditionVariable,
                          IN OUT PCRITICAL_SECTION CriticalSection,
                          IN DWORD dwMilliseconds)
 {
-    NTSTATUS Status = 0;
+    NTSTATUS Status = STATUS_SUCCESS;
+#if 0
```

Comment:
I understand that this will avoid "unused variable" warnings -- but why is this influenced by "compiling as NT6"?

Reply:
See lines 15 and 718.
Assumed somewhat related to CORE-14915...

Reply:
No, this is a bad way. Better add RtlSleepConditionVariableCS/RtlSleepConditionVariableSRW to export list of ntdll_vista and add it to kernel32's add_importlibs